Dip/Dip/Dip.swift

```diff
@@ -186,28 +186,30 @@ public final class DependencyContainer {
    Though before you do that you should probably review your design and try to reduce the number of dependencies.
    
    */
-  public func resolve<T, F>(tag tag: Tag? = nil, builder: F->T) throws -> T {
+  public func resolve<T, F>(tag tag: Tag? = nil, builder: F throws -> T) throws -> T {
```

Comment:
Not sure if the outer `throws` should be `throws` or `rethrows`?

http://robnapier.net/re-throws